### PR TITLE
Enhacements

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | https://github.com/yiisoft/yii-db-migration/issues/18

- [x] change view for webview (always defined in the application).
- [x] add final class create migrations.
- [x] rework aliases.
- [x] Set of template paths for generating migration code automatically.

### CODE REVIEW

File: config/console.php [hunk 1]
```diff
@@ -2,5 +2,11 @@
 
 declare(strict_types=1);
 
+use Yiisoft\Aliases\Aliases;
+
 return [
+    /** ALIASES */
```
Comment (samdark) [inline config/console.php:8]: It's clear that it is aliases without this comment.

File: src/Service/Generate/CreateService.php [hunk 2]
```diff
@@ -32,20 +32,20 @@ final class CreateService
     private Connection $db;
     private ConsoleHelper $consoleHelper;
     private MigrationService $migrationService;
-    private View $view;
+    private WebView $webView;
```
Comment (samdark) [inline src/Service/Generate/CreateService.php:35]: Why changing it?

Comment (terabytesoftw) [reply src/Service/Generate/CreateService.php:35]: In the application we define webview, no view, simple configuration.

File: config/console.php (hunk 1 above)
Comment (terabytesoftw) [reply config/console.php:8]: fixed.

**Pull request merged**